src/content/en/fundamentals/app-install-banners/index.md

```diff
@@ -148,7 +144,7 @@ To determine if the app was successfully added to the user's home screen _after_
 they accepted the prompt, you can listen for the `appinstalled` event.
 
     window.addEventListener('appinstalled', (evt) => {
-      app.logEvent('a2hs', 'installed');
+      console.log('a2hs installed');
```

Comment:
It seems that the intent here was to log the `a2hs: installed` event with the intent to send it to analytics or to the server. If you want to change this to be a simple `console.log`, it think it would be better to say `console.log('app installed')`